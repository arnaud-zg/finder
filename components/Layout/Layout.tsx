import React from 'react'
import { style } from 'typestyle'
import { Navigation } from '../Navigation'

const className = style({
  padding: '0 32px',
})

export class Layout extends React.PureComponent {
  render() {
    const { children } = this.props
    return (
      <>
        <Navigation />
        <div className={className}>{children}</div>
      </>
    )
  }
}
