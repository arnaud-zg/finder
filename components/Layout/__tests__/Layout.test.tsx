import React from 'react'
import renderer from 'react-test-renderer'
import { Layout } from '../Layout'

describe('Layout', () => {
  it('renders correctly', () => {
    const testRenderer = renderer.create(<Layout />)
    expect(testRenderer.toJSON()).toMatchSnapshot()
  })
})
