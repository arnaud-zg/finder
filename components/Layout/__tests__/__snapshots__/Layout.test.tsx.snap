// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders correctly 1`] = `
Array [
  <nav
    className="f1lqund8"
  >
    <ul>
      <li>
        <a
          href="/"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          Home
        </a>
      </li>
      <li>
        <a
          href="/announcement"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          Announcement
        </a>
      </li>
      <li>
        <a
          href="/about"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          About
        </a>
      </li>
    </ul>
  </nav>,
  <div
    className="f1kip1e6"
  />,
]
`;
